### PR TITLE
chef syndi whetstone

### PR DESCRIPTION
**Pull request by TheFakeElon**
## About The Pull Request

adds the "bluespace whetstone" a sharpening block that increases the range on almost any sharp weapon

## Why It's Good For The Game

chef buff uwu

## Changelog
:cl:
add: Traitor chef whetstone
/:cl:

### CODE REVIEW

**Comment by TheFakeElon** [comment]
happy to receive suggestions regarding price.

**Comment by thelaughingbomb** [comment]
This + poison knife 😳

**Comment by MCterra10** [comment]
nice I like

File: austation/code/game/objects/items/sharpener.dm [hunk 1]
```diff
@@ -0,0 +1,50 @@
+/obj/item/sharpener/bluespace
+	name = "bluespace whetstone"
+	desc = "A bluespace block that can enhance your weapon, allowing it to phase through space."
+	icon = 'austation/icons/obj/kitchen.dmi'
+	icon_state = "bsharpener"
+	increment = 10
+	max = 50
+	prefix = "bluespace sharpened"
+	requires_sharpness = 0
+	var/range_increment = 1 // amount to increase range by
```
Comment (MCterra10) [inline austation/code/game/objects/items/sharpener.dm:10]: based, can use this for admemes, thanks elon

Comment (TheFakeElon) [reply austation/code/game/objects/items/sharpener.dm:10]: hahaha, you're welcome <3

**Comment by MCterra10** [comment]
whoops, habit

**Pull request merged**